account-kit/rn-signer/src/utils/parseMfaError.ts

```diff
@@ -0,0 +1,15 @@
+import type { MfaFactor } from "@account-kit/signer";
+
+export function parseMfaError(error: unknown): MfaFactor[] | null {
+  if (error instanceof Error) {
+    try {
+      const parsed = JSON.parse(error.message);
+      if (parsed?.data?.multiFactors) {
+        return parsed.data.multiFactors;
+      }
+    } catch {
+      // ignore JSON parse failures
```

Comment:
❓ Why are we ignoring it, I guess if we want to debug this will be findable in the network right?

Reply:
Yeah good question, json.parse throws an error if you pass it invalid json data.  I am assuming here if we are passing invalid json, it's not an mfa error.  In that case, we should return null and let the calling function handle the error